Sources/SpeziContact/Models/ContactOption.swift

```diff
@@ -11,6 +11,18 @@ import SwiftUI
 
 
 /// Customizable way to get in contact with an individual and usually connected to a ``Contact``.
+///
+/// The following example demonstrates the usage of the `Contact`.
+/// ```swift
+/// ContactOption(
```

Comment:
Similar to above, it would be great to not just show the initializer but embed this within a slightly larger example that demonstrates the usage